dist/s_define.list

```diff
@@ -40,6 +40,7 @@ WT_LOG_V3_MINOR
 WT_LOOKASIDE_COMPRESSOR
 WT_OPTRACK_BUFSIZE
 WT_OPTRACK_MAXRECS
+WT_ORDERED_WRITE
```

Comment:
Added `WT_ORDERED_WRITE` to the igone list `s_define.list` because it is being used only once in `src/reconcile/rec_visibility.c` and `s_define` was reporting about it.